src/ui/ui_settings/__tests__/lib/call_cluster_stub.js

```diff
@@ -0,0 +1,48 @@
+import sinon from 'sinon';
+import expect from 'expect.js';
+import { merge } from 'lodash';
+
+export function createCallClusterStub(index, type, id, esDocSource) {
+  const currentEsDocValue = merge({}, esDocSource);
+
+  const callCluster = sinon.spy(async (method, params) => {
+    expect(params)
```

Comment:
This method would work fine without the state, so I'll remove it, but do you know if the `sinon.stub()` that has multiple `withArgs()` branches will throw if calls don't match the stubbed arg tests? Also, I didn't realize you could do that you did with `sinon.match()`... I'm all for less stub logic but think that implementing a function inside of `sinon.spy()` can be a bit easier to read and understand (especially since I don't understand sinon inside an out)